gulpfile.js

```diff
@@ -0,0 +1,53 @@
+var _ = require('underscore');
+var fs = require('fs');
+var gulp = require('gulp');
+var consolidate = require('gulp-consolidate');
+var rename = require('gulp-rename');
+var svgmin = require('gulp-svgmin');
+var urlencode = require('gulp-css-urlencode-inline-svgs');
+
+gulp.task('minify-icons', function() {
+  return gulp.src('./fec-icons/icons/*.svg')
+    .pipe(svgmin({
+      plugins: [
+        {
+          removeDimensions: true
+        },
+        {
+          removeAttrs: {attrs: '(fill|fill-rule)'}
+        }
+      ]
+    }))
+    .pipe(gulp.dest('./img/icons', {overwrite: true}));
+});
+
+gulp.task('consolidate-icons', function() {
+  function getSVGs() {
```

Comment:
This is just me, but I like to chain things together with underscore. It might look like this:
```javascript
function getSVGs() {
  return _(fs.readdirSync('./img/icons/'))
    .chain()
    .filter(function (filename) {
      return filename.substr(-4) === '.svg';
    }).map(function (filename) {
      return {
        name: filename.split('.')[0],
        content: fs.readFileSync('./img/icons/' + filename, 'utf8')
      };
    }).value();
}
```